serf/.dockerignore

```diff
@@ -1 +1,2 @@
 *
+!install-tools
```

Comment:
Because an [older PR](https://github.com/cybozu/neco-containers/commit/c6f453b02dd4265ac5d63de810e94d4d740b3507) changes these files:
- `kubernetes/Dockerfile`
- `kubernetes/install-tools`
- `serf/.dockerignore`.

The serf configuration has been broken since the PR, but it did not cause any effect because it did not update serf TAG.

Reply:
Oh, what a...